pybv/io.py

```diff
@@ -59,20 +59,18 @@ def write_brainvision(*, data, sfreq, ch_names, fname_base, folder_out,
         Currently all events are written as type "Stimulus" and must be
         numeric.
         Defaults to None (not writing any events).
-    resolution : float | ndarray
-        The resolution **in volts** in which you'd like the data to be stored.
-        By default, this will be 1e-7, or 0.1 µV. This number controls the
-        amount of round-trip resolution. This can be either a single float for
-        all channels or an array with n_channels elements.
-    unit : str | None
+    resolution : float | ndarray, shape(nchannels,)
+        The resolution in `unit` in which you'd like the data to be stored.
+        If float, the same resolution is applied to all channels.
+        If ndarray with n_channels elements, each channel is scaled with
+        its own corresponding resolution from the ndarray.
+        Note that if `fmt` is 'binary_int16', it is recommended to set
+        `resolution` to '0.1' (default) and `unit` to 'µV' (default).
```

Comment:
actually, "binary_int16" shouldn't be used at all until we figure out why roundtrips are failing

Reply:
I figured it out now, will push a follow up PR after this one.